### ISSUE TITLE
[Fluent UI- Send box: attaching a file]: Screen reader does not announce "sending" while uploading video file.

### ISSUE THREAD

**Issue by Kusum199** [microsoft/BotFramework-WebChat#5700]
### What is the PWD impact?

Screen reader users are not informed that the video upload is in progress, causing confusion, repeated actions, and uncertainty about whether the file is being sent successfully.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Send box

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3.Navigate to Fluent UI experience chat page and attach a video file after selecting attachment icon and then send it.
4. Observe that screen reader dose not announce "sending" while uploading the video file.

### What do you expect?

When uploading a video file, the screen reader should announce “Sending” (or an equivalent status message) to inform users that the upload is in progress.

### What actually happened?

Screen reader does not announce "sending" while uploading video file.

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/6319f2b5-ca7f-486c-ae22-46e4c47d549c

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

### Adaptive Card JSON

```json

```
